tests/PostApiTest.php

```diff
@@ -5,6 +5,8 @@
 
 class PostApiTest extends TestCase
 {
+    use WithoutMiddleware;
```

Comment:
Not necessary for merge, but I'd recommend making a helper to auth individual requests (like a `$this->authed()->json(...)` rather than disabling middleware across the board, since that allows the test environment to diverge a lot from how the real application works.